pep-0683.rst

```diff
@@ -0,0 +1,335 @@
+PEP: 683
+Title: Immortal Objects, Using a Fixed Refcount
+Author: Eric Snow <ericsnowcurrently@gmail.com>, Eddie Elizondo <eduardo.elizondorueda@gmail.com>
+Discussions-To: python-dev@python.org
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 10-Feb-2022
+Python-Version: 3.11
+Post-History:
+Resolution:
+
+
+Abstract
+========
+
+Under this proposal, any object may be marked as immortal.
+"Immortal" means the object will never be cleaned up (at least until
+runtime finalization).  Specifically, the `refcount`_ for an immortal
+object is set to a sentinel value, and that refcount is never
+changed by ``Py_INCREF()`` or ``Py_DECREF()``.
```

Comment:
`Py_SET_REFCNT`  won't change the refcount either! https://github.com/python/cpython/pull/19474/files#r776528822